src/pages/auth/gravatar.mdx

```diff
@@ -0,0 +1,7 @@
+---
+title: "Gravatar"
+---
+
+Gravatar(Globaly Recognized Avatar) is a service for providing globally unique avatars by Wordpress.
+
+By default, every new account that is registered is checked against Gravatar. If there exists an account with the same email on Gravatar, we get the avatar from there.
```

Comment:
```suggestion
By default, every new account that is registered is checked against Gravatar. If there exists an account with the same email address on Gravatar, the already existing avatar is retrieved from there.
```

Reply:
Thanks @johannesschobel  for your reviews. We're waiting with this a bit since we're working hard on our new infrastructure that will use Hasura Auth [WIP] (instead of Hasura Backend Plus) which will include this update.

Things are a bit chaotic at the moment :>

Reply:
Dear @elitan , thanks a lot for your response. I was just browsing some PRs and thought i can provide feedback ;) So don't worry.
Will you move to Hasura2 also?
All the best

Reply:
Yea your comments are very welcomed! Appreciate it.

Yes, we'll move to Hasura v2 👍

Reply:
that is awesome. do you have an estimated timeplan? Thinking about using nhost for the next project at university ;)

Reply:
We'll start with a private beta at the end of August. If you have an account on Nhost we'll send out an email to you once things open up generally.